### PR TITLE
improve lambda err

### LINKED ISSUE #81: Improve the error handling of the POST API
Right now, the POST API doesn't handle improper input (like empty input or non JSON input that is valid). We need to improve the error handling for those conditions. Here are sample requests that you can use for testing:

```
curl -X POST -d "" -H "Content-Type: application/json" -v https://p9y9bbcruj.execute-api.us-west-2.amazonaws.com/default/simplePostLambda
curl -X POST -d "asdfasdfasdfasfd" -H "Content-Type: application/json" -v https://p9y9bbcruj.execute-api.us-west-2.amazonaws.com/default/simplePostLambda
  curl -X POST -d "{\"someprop\":\"someval\"}" -H "Content-Type: application/json" -v https://p9y9bbcruj.execute-api.us-west-2.amazonaws.com/default/simplePostLambda
```

### PR DESCRIPTION
**Pull request by TurkiBK**
close #81 
### User Story 
Lambda accept teacher feedback ,and only return the feedback and successful invoke with 201.It event or feedback is not empty , or missing on syntax , it will return error message.

### To test it

Prerequisites
 Terminal 
Postmen  
Endpoint : https://p9y9bbcruj.execute-api.us-west-2.amazonaws.com/default/simplePostLambda

### Using postmen or smiler

1. Copy the endpoint and post it on the postmen url  

2. Change method from get to post
3. select body, raw ,and change text to JSON
4. sending json as {"feedback":"write something here "}
5. click send
6. getting response with 201 when it is succeed.
7. getting response 400 for empty string

### Using terminal

1.  curl -d '{"feedback":"Good job!"}' -H "Content-Type: application/json" -X POST https://p9y9bbcruj.execute-api.us-west-2.amazonaws.com/default/simplePostLambda
2. getting response with 201 when it is succeed
3. getting response 400 for empty string

###  Gallery
here some picture when feedback return  succeed or errors.

Screen Shot 2022-02-28 at 6 39 47 PM

Screen Shot 2022-02-28 at 6 39 18 PM

Screen Shot 2022-02-28 at 6 38 36 PM

### Estimated time

Screen Shot 2022-02-28 at 6 37 09 PM

### CODE REVIEW

File: backend/simpleLambda/index.js [hunk 1]
```diff
@@ -1,51 +1,52 @@
-const messafe='null'
+exports.handler = async (event) => {
+  console.log("event: " + JSON.stringify(event));
+  console.log("----- lambda start");
```
Comment (CalebMcOlin) [inline backend/simpleLambda/index.js]: Do you need all these console logs still?  Typically you'd want to remove all console logs before pushing into a main branch. _(there are two more on lines 8 and 9)_

**Review by CalebMcOlin — APPROVED** [review]
Tested and everything worked as intended! Nice work.

**Review by KristinJJ — APPROVED** [review]
API responds as intended following testing in Postman and cURL, PR approved!

**Comment by KristinJJ** [comment]
Also in addition to what Caleb mentioned above about removing console.logs, just a reminder about this part of the Syllabus under Task Management for full credit:
```
·      Task must be created and appear on the project board
·      Task must be tagged with the proper labels (team and bug/task)
·      Task must be tagged with the proper milestone
·      Task must be assigned to the responsible developer
·      Task must be in In Progress state while in development
·      Task must be in Done state at the end of the sprint
```

**Review by KristinJJ — CHANGES_REQUESTED** [review]
Hi Turki, I saw there was a merge conflict due to you and I both having worked on the simpleLambda folder. I was reviewing the recording from week7, and it seems like for Sprint 3 you are supposed to have made your changes to index.js in a copy of the simpleLambda folder, called simpleLambda01 to prevent merge conflicts for Sprint 3. If you move your changed index.js file to a new folder called simpleLambda01, I will go ahead and re-approve this PR!

File: backend/simpleLambda/index.js (hunk 1 above)
Comment (TurkiBK) [reply backend/simpleLambda/index.js]: No, I am gonna clean the code .

**Pull request closed without merging**